//Exception  Handling

try{
  num=prompt("Enter  a number ")
if(num===' ')
  throw'cannot be empty'
if(isNaN(num))
throw"Enter a Valid Number"
 console.log(num**2)

}
catch(error){
  console.log(error)
}
finally{
  console.log('bye')
}